domain-mapping/cli/class-darkmatter-domain-cli.php

```diff
@@ -63,16 +70,30 @@ public function add( $args, $assoc_args ) {
 			[
 				'disable' => false,
 				'force'   => false,
-				'https'   => false,
+				'https'   => true,
 				'primary' => false,
+				'type'    => false,
 			]
 		);
 
+		/**
+		 * Handle the Media flag.
+		 */
+		$domain_types = [
+			'main'  => DM_DOMAIN_TYPE_MAIN,
+			'media' => DM_DOMAIN_TYPE_MEDIA,
+		];
+		$type         = DM_DOMAIN_TYPE_MAIN;
+
+		if ( array_key_exists( $opts['type'], $domain_types ) ) {
```

Comment:
I'd lowercase `$opts['type']`

Reply:
Good suggestion - done this in the new method in response to the later note for duplicate code.